pygments/lexers/ride.py

```diff
@@ -48,26 +48,34 @@ class RideLexer(RegexLexer):
         'WriteSet', 'AttachedPayment', 'ScriptTransfer', 'TransferSet',
         'ScriptResult', 'Invocation', 'Asset', 'BlockInfo', 'Issue', 'Reissue',
         'Burn', 'NOALG', 'MD5', 'SHA1', 'SHA224', 'SHA256', 'SHA384', 'SHA512',
-        'SHA3224', 'SHA3256', 'SHA3384', 'SHA3512',
+        'SHA3224', 'SHA3256', 'SHA3384', 'SHA3512', 'BinaryEntry',
+        'BooleanEntry' , 'IntEntry', 'StringEntry', 'List', 'Ceiling',
+        'Down', 'Floor', 'HalfDown', 'HalfEven', 'HalfUp', 'Up',
     )
 
     functionsName = (
         'fraction', 'size', 'toBytes', 'take', 'drop', 'takeRight', 'dropRight',
         'toString', 'isDefined', 'extract', 'throw', 'getElement', 'value',
-        'cons', 'ensure', 'toUtf8String', 'toInt', 'indexOf', 'split',
+        'cons', 'toUtf8String', 'toInt', 'indexOf', 'lastIndexOf', 'split',
         'parseInt', 'parseIntValue', 'keccak256', 'blake2b256', 'sha256',
         'sigVerify', 'toBase58String', 'fromBase58String', 'toBase64String',
         'fromBase64String', 'transactionById', 'transactionHeightById',
         'getInteger', 'getBoolean', 'getBinary', 'getString',
         'addressFromPublicKey', 'addressFromString', 'addressFromRecipient',
         'assetBalance', 'wavesBalance', 'getIntegerValue', 'getBooleanValue',
         'getBinaryValue', 'getStringValue', 'addressFromStringValue',
-        'assetInfo', 'rsaVerify', 'checkMerkleProof',
+        'assetInfo', 'rsaVerify', 'checkMerkleProof', 'median',
+        'valueOrElse', 'valueOrErrorMessage', 'contains', 'log', 'pow',
+        'toBase16String', 'fromBase16String', 'blockInfoByHeight',
+        'transferTransactionById', 'nil', 'this', 'unit', 'height',
+        'lastBlock', 'Buy', 'Sell', 'CEILING', 'FLOOR', 'DOWN', 'HALFDOWN',
+        'HALFEVEN', 'HALFUP', 'UP', 'NOALG', 'MD5', 'SHA1', 'SHA224','SHA256',
+        'SHA384', 'SHA512', 'SHA3224', 'SHA3256', 'SHA3384', 'SHA3512',
```

Comment:
Some of those are now doubled in `typesName` and `functionsName`, e.g. `NOALG`...`SHA3384`. Since types are first, they will always be highlighted as such.

`HalfDown` etc. are also doubled, but with different casing...

Reply:
Thanks for your attention! Fixed